docs/tutorials/developer-journey/level-1/1.6-managing-canisters.md

```diff
@@ -0,0 +1,259 @@
+
+To check the current state of all canisters, you can use the command:
+
+```
+dfx canister status --network ic --all
+```
+
+Or, you can check the status of a single canister by specifying it's name, such as:
+
+```
+dfx canister status poll_backend --network ic 
+```
+
+The output of the command will resemble the following:
+
+```
+Canister status call result for poll_backend.
+Status: Running
+Controllers: lalyb-uhvmt-p7ubs-u5t7l-hce6v-lp7c5-dlmj5-wi2gc-depab-wtgi3-pae
+Memory allocation: 0
+Compute allocation: 0
+Freezing threshold: 2_592_000
+Memory Size: Nat(2363181)
+Balance: 3_100_000_000_000 Cycles
+Module hash: 0xf8680eb74022a1079012b7e9c644d1156580002a6126305791811533d3fd6f17
+```
+
+Stopping a canister works in a similar fashion. For example, to stop a single canister, run the command:
+
+```
+dfx canister stop poll_backend --network ic 
+```
+
+Or, to stop all canisters, use the command:
+
+```
+dfx canister stop --network ic --all
+```
+
+This command displays output similar to the following:
+
+```
+Stopping code for canister poll_backend, with canister_id 5o6tz-saaaa-aaaaa-qaacq-cai
+Stopping code for canister poll_frontend, with canister_id 5h5yf-eiaaa-aaaaa-qaada-cai
+```
+
+Then you can verify that the canisters have been stopped by rerunning the `dfx canister status` command:
+
+```
+dfx canister status --network ic poll_backend
+```
+
+Which should return:
+
+```
+Canister status call result for poll_backend.
+Status: Stopped
+Controllers: lalyb-uhvmt-p7ubs-u5t7l-hce6v-lp7c5-dlmj5-wi2gc-depab-wtgi3-pae
+Memory allocation: 0
+Compute allocation: 0
+Freezing threshold: 2_592_000
+Memory Size: Nat(2363181)
+Balance: 3_100_000_000_000 Cycles
+Module hash: 0xf8680eb74022a1079012b7e9c644d1156580002a6126305791811533d3fd6f17
+```
+
+Then to start the canister again, run the command:
+
+```
+dfx canister start --network ic --all
+```
+
+This will return a confirmation such as:
+
+```
+Starting code for canister poll_backend, with canister_id 5o6tz-saaaa-aaaaa-qaacq-cai
+Starting code for canister poll_frontend, with canister_id 5h5yf-eiaaa-aaaaa-qaada-cai
+```
+
+## Setting the canister's freezing threshold
+
+In the output of the `dfx canister status` command, you may have noticed the `freezing threshold` value. This freezing threshold refers to the amount of cycles that a canister will not be allowed to dip below. This is because if a canister runs out of cycles, it will be uninstalled. The freezing threshold protects it from deletion, since if the cycles balance dips below the threshold, the canister will stop processing any new requests; however, it will continue to reply to existing requests. 
```

Comment:
I think this needs a bit more information

The freezing threshold is actually defined in seconds and not in cycles. The default value `2_592_000s ` corresponds to 30 days. The actual threshold in cycles is calculated by the system and depends on the memory consumption of the canister. As soon as a call to a canister leads to a dip below the respective threshold, the caller will get an error.